packages/cli/src/lib/config.ts

```diff
@@ -44,10 +46,22 @@ async function createConfig(prompt: Prompt = new InquirerPrompt()): Promise<Conf
     const jdkInstaller = new JdkInstaller(process);
     jdkPath = await jdkInstaller.install(DEFAULT_JDK_FOLDER);
   }
-  const androidSdkPath = await prompt.promptInput('Path to the Android SDK:', null,
-      AndroidSdkTools.validatePath);
 
-  return new Config(jdkPath, androidSdkPath);
+  const sdkInstallRequest = await prompt.promptConfirm('Do you want Bubblewrap to install ' +
+    'Android SDK? (Enter "No" to use your installation)', true);
+
+  let sdkPath;
+  if (!sdkInstallRequest) {
+    sdkPath = await prompt.promptInput('Path to your existing Android SDK:', null,
+        AndroidSdkTools.validatePath);
+  } else {
+    await fsPromises.mkdir(DEFAULT_SDK_FOLDER);
+    console.log(`Downloading Android command line tools to ${DEFAULT_SDK_FOLDER}`);
```

Comment:
Isn't the SDK more than just the command line tools?

Reply:
The download page says "Command line tools only" when downloading it, so I'm ok with calling it "Command line tools". But we should make the name consistent (we use Android SDK on other places).

Reply:
I just changed everything to "Android SDK" everywhere to be consistent.